"out_of_memory" redefined

From a clean go-get:

```sh
$ go get github.com/wellington/go-libsass/
# github.com/wellington/go-libsass/libs
In file included from unity.cpp:22:0:
../libsass-build/json.cpp:49:0: warning: "out_of_memory" redefined
 #define out_of_memory() do {                    \
 
In file included from ../libsass-build/ast.hpp:52:0,
                 from ../libsass-build/ast.cpp:2,
                 from unity.cpp:4:
../libsass-build/util.hpp:15:0: note: this is the location of the previous definition
   #define out_of_memory() do {            \
```

So I tried pulling down 3.5 which would include this build. I'm getting some nasty errors though

```
In file included from cencode.c:2:
libs/../libsass-build/cencode.c:50:5: warning: declaration does not declare anything [-Wmissing-declarations]
libs/../libsass-build/cencode.c:64:5: warning: declaration does not declare anything [-Wmissing-declarations]
# github.com/wellington/go-libsass/libs
Undefined symbols for architecture x86_64:
  "Sass::traces_to_string(std::__1::vector<Sass::Backtrace, std::__1::allocator<Sass::Backtrace> >, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >)", re
ferenced from:
      Sass::Eval::operator()(Sass::Warning*) in unity.cpp.o
      Sass::handle_error(Sass_Context*) in unity.cpp.o
  "Sass::Operators::op_numbers(Sass_OP, Sass::Number const&, Sass::Number const&, Sass_Inspect_Options, Sass::ParserState const&, bool)", referenced from:
      Sass::Eval::operator()(Sass::Binary_Expression*) in unity.cpp.o
      _sass_value_op in unity.cpp.o
```

I have mostly kept away from mutating the libsass source in go-libsass, but that may be necessary to get around this build error for now.

--- a/Makefile
+++ b/Makefile
@@ -46,7 +46,7 @@ update-libsass: libsass-build libsass-build/include/sass/version.h
 
 .PHONY: test
 test:
-	go test -tags dev -race .
+	go test -race .
 
 cleanfiles:
 	rm -rf lib include libsass-src libsass-tmp

--- a/libsass-build/sass.cpp
+++ b/libsass-build/sass.cpp
@@ -33,8 +33,10 @@ extern "C" {
   void* ADDCALL sass_alloc_memory(size_t size)
   {
     void* ptr = malloc(size);
-    if (ptr == NULL)
-      out_of_memory();
+    if (ptr == NULL) {
+      std::cerr << "Out of memory.\n";
+      exit(EXIT_FAILURE);
+    }
     return ptr;
   }
 

--- a/libsass-build/util.hpp
+++ b/libsass-build/util.hpp
@@ -12,10 +12,11 @@
 
 namespace Sass {
 
-  #define out_of_memory() do {            \
-      std::cerr << "Out of memory.\n";    \
-      exit(EXIT_FAILURE);                 \
-    } while (0)
+  // This will throw errors if built, please keep this as is
+  // #define out_of_memory() do {            \
+  //     std::cerr << "Out of memory.\n";    \
+  //     exit(EXIT_FAILURE);                 \
+  //   } while (0)
 
   double round(double val, size_t precision = 0);
   double sass_strtod(const char* str);

